libs/gltfio/CMakeLists.txt

```diff
@@ -120,8 +120,12 @@ build_ubershader(specularGlossiness_masked base            specularGlossiness ma
 build_ubershader(unlit_fade                base            unlit fade)
 build_ubershader(unlit_opaque              base            unlit opaque)
 build_ubershader(unlit_masked              base            unlit masked)
-build_ubershader(volume                    volume          _ _)
-build_ubershader(transmission              transmission    _ _)
+build_ubershader(volume_opaque             volume          lit opaque)
+build_ubershader(volume_fade               volume          lit fade)
+build_ubershader(volume_masked             volume          lit masked)
+build_ubershader(transmission_opaque       transmission    lit opaque)
+build_ubershader(transmission_fade         transmission    lit fade)
+build_ubershader(transmission_masked       transmission    lit masked)
```

Comment:
so that's going to increase the size of the gltfio a lot right? Is that what we want? I suppose we have no choice?

Reply:
No choice unfortunately :/